### ISSUE TITLE
Crashes after import of LoyaltyCardManager

### ISSUE THREAD

**Issue by oscaropenness** [TheLastProject/Catima#100]
### Steps to reproduce
1. export data from Loyalty Card Keychain to .csv
```
_id,store,note,cardid,headercolor,headertextcolor,barcodetype
4,Shop1,,13478392874,-7617718,-1,ITF
1,Shop2,,123789012345,,,ITF
5,Shop3,,1285647320198,-957596,-1,EAN_13
2,Shop4,,S74623490,,,CODE_39
3,Shüp.-,7845,47322349175647382,-686759,-1,CODE_128
6,Shop,XZW jspwer,K1412223,-14642227,-1,CODE_128
```
2. import .csv into Catima

### Expected Result:
cards get imported into Catima

### Actual Result:
Message saying cards were imported
After that Catima crashes

```FATAL EXCEPTION: main
Process: me.hackerchick.catima, PID: 2266
java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.Integer.intValue()' on a null object reference
	at protect.card_locker.Utils.needsDarkForeground(Utils.java:24)
	at protect.card_locker.Utils.generateIcon(Utils.java:20)
	at protect.card_locker.LoyaltyCardCursorAdapter.bindView(LoyaltyCardCursorAdapter.java:74)
	at android.widget.CursorAdapter.getView(CursorAdapter.java:296)
	at android.widget.AbsListView.obtainView(AbsListView.java:2405)
	at android.widget.ListView.measureHeightOfChildren(ListView.java:1417)
	at android.widget.ListView.onMeasure(ListView.java:1323)
	at android.view.View.measure(View.java:25086)
	at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:735)
	at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:481)
	at android.view.View.measure(View.java:25086)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6872)
	at androidx.coordinatorlayout.widget.CoordinatorLayout.onMeasureChild(CoordinatorLayout.java:760)
	at com.google.android.material.appbar.HeaderScrollingViewBehavior.onMeasureChild(HeaderScrollingViewBehavior.java:99)
	at com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior.onMeasureChild(AppBarLayout.java:1996)
	at androidx.coordinatorlayout.widget.CoordinatorLayout.onMeasure(CoordinatorLayout.java:831)
	at android.view.View.measure(View.java:25086)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6872)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at androidx.appcompat.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:146)
	at android.view.View.measure(View.java:25086)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6872)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1552)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:842)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:721)
	at android.view.View.measure(View.java:25086)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6872)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at android.view.View.measure(View.java:25086)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6872)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1552)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:842)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:721)
	at android.view.View.measure(View.java:25086)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6872)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at com.android.internal.policy.DecorView.onMeasure(DecorView.java:742)
	at android.view.View.measure(View.java:25086)
	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:3083)
	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1857)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2146)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1745)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7768)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:967)
	at android.view.Choreographer.doCallbacks(Choreographer.java:791)
	at android.view.Choreographer.doFrame(Choreographer.java:726)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:952)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:940)
```

### System
LineageOS 17.1

I first suspected the umlaut in one of my shopnames, but removing that did not solve the issue?
Any ideas, what the problem is?

**Comment by TheLastProject** [comment]
Thank you for the example export, I can reproduce the crash with it.

The issue here is that some cards have no background colour set at all. Didn't know that was a thing that could happen but I will fix this crash soon.

**Comment by oscaropenness** [comment]
Thanks for the quick response!
I suspected something to be wrong with the export, but that's what Loyalty Card Keychain produced.

Thank you for looking into it!

**Comment by TheLastProject** [comment]
Fixed in 67b961c6728d465f4485aca2f42ad65291e52e5e and released in 1.2.0: https://github.com/TheLastProject/Catima/releases/tag/v1.2.0. Thanks again for the detailed report!
